SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Hyperledger Security Policy
+
+## Reporting a Security Bug
+
+If you think you have discovered a security issue in any of the Hyperledger projects, we'd love to
+hear from you. We will take all security bugs seriously and if confirmed upon investigation we will
```

Comment:
```suggestion
hear from you. We take all security bugs seriously and if confirmed upon investigation we will
```

Reply:
This file actually needs to have the same contents as other Hyperledger repos, so this was just a copy of what they have.